knox/views.py

```diff
@@ -60,7 +60,7 @@ def post(self, request, format=None):
                     status=status.HTTP_403_FORBIDDEN
                 )
         token_ttl = self.get_token_ttl()
-        instance, token = AuthToken.objects.create(request.user, token_ttl)
+        instance, token = get_token_model().objects.create(request.user, token_ttl)
```

Comment:
As per my comment above, this still implies some constraint on how the custom AuthToken model would work as it needs to comply to what happens in the create method and also reuse the manager from the default model.

Reply:
That or we could simply have a `create_token(request)` method.